hw2/04-background.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>04 - Background</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="04-background.js"></script>
+  </head>
+  <body class="bg-danger-subtle">
+    <main class="w-50 mx-auto border mt-5 p-3">
+      <h1>Exercise 04 - Background</h1>
+      <p>
+        The background color will change every few seconds (as indicated below).
+        You can stop this behavior and you can also change how often the
+        background color changes.
+      </p>
+      <div class="mb-4">
+        <label class="form-label">Background change interval (seconds): </label>
```

Comment:
Here I get a WAVE audit error, looks like the label needs the "for" attribute set match the "id" attribute of the input tag. Exercise 4 works great besides that